src/components/feature/settings/FileManagement/index.tsx

```
@@ -0,0 +1,63 @@
+  };
+};
+
+export const FileManagement = ({ currentUser }: FileManagementProps) => {
+  const [files, setFiles] = useState<File[]>([]);
+
+  useEffect(() => {
+    const fetchFiles = async () => {
+      try {
+        const fetchedFiles = await getFiles();
+        setFiles(fetchedFiles);
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)